Sources/DbSearch-postgresql.php

```diff
@@ -91,11 +105,16 @@ function smf_db_search_query($identifier, $db_string, $db_values = array(), $con
 
 	if (isset($replacements[$identifier]))
 		$db_string = preg_replace(array_keys($replacements[$identifier]), array_values($replacements[$identifier]), $db_string);
-	elseif (preg_match('~^\s*INSERT\sIGNORE~i', $db_string) != 0)
+	if (preg_match('~^\s*INSERT\sIGNORE~i', $db_string) != 0)
```

Comment:
I don't understand the intention,
why here is a elseif used.

I got some error around this,
so i replace this with a if.